terraform/amazon/network-existing-vpc/igw.tf

```diff
@@ -0,0 +1,10 @@
+resource "aws_internet_gateway" "main" {
```

Comment:
I think you can only have a single IGW per VPC, so you shouldn't create a new one

Reply:
Removed this file